{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env sh
 {% include 'misc/header.py' %}
 
-pydocstyle {{ cookiecutter.package_name }} && \
+pydocstyle {{ cookiecutter.package_name }} tests && \
 isort -rc -c -df **/*.py && \
-check-manifest --ignore ".travis-*" && \
+check-manifest --ignore ".travis-*,docs/_build" && \
```

Comment:
I'm not sure if it's the good place since the `.travis-*` files are only generated on Travis CI.